doc/rados/operations/health-checks.rst

```diff
@@ -870,25 +872,26 @@ The state of specific problematic PGs can be queried with:
 PG_DEGRADED
 ___________
 
-Data redundancy is reduced for some data, meaning the cluster does not
-have the desired number of replicas for all data (for replicated
-pools) or erasure code fragments (for erasure coded pools).
-Specifically, one or more PGs:
+Data redundancy is reduced for some data: in other words, the cluster does not
+have the desired number of replicas for all data (in the case of replicated
+pools) or erasure code fragments (in the case of erasure-coded pools).  More
+precisely, one or more Placement Groups (PGs):
 
-* has the *degraded* or *undersized* flag set, meaning there are not
-  enough instances of that placement group in the cluster;
-* has not had the *clean* flag set for some time.
+* have the *degraded* or *undersized* flag set, which means that there are not
+  enough instances of that PG in the cluster; or
+* have not had the *clean* flag set for a long time.
```

Comment:
Suggest s/flag/state/

Reply:
unfinished business--flags are part of the process of setting states, and this very thing is alluded to just above. We'll return to this later.